database/02-Roles-Table-Test-Data.sql

```diff
@@ -0,0 +1,2 @@
+USE Team1_KelvinC;
+INSERT INTO Roles (roleId, capability, jobFamily, jobProfileTitle, managementLevel, description, minimalEssentialRequirements) VALUES (1, 'test','test','test','test','test','test');
```

Comment:
Could you add a new line to this or you could merge my current branch with yours because I've updated this.